mmv1/third_party/terraform/provider/provider_credentials_test.go

```diff
@@ -0,0 +1,263 @@
+package provider_test
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+// TestAccSdkProvider_credentials is a series of acc tests asserting how the SDK provider handles credentials arguments
+// It is SDK specific because the HCL used provisions SDK-implemented resources
+// It is a counterpart to TestAccFwProvider_credentials
+func TestAccSdkProvider_credentials(t *testing.T) {
```

Comment:
This file was made by copy-pasting `framework_provider_credentials_test.go` and updating the HCL to be SDK specific. We could reduce this duplication by combining in one file, but it would be really easy to accidentally make tests that pass that don't actually test the SDK and the PF (that all depends on the HCL used). I'm open to ideas!